classes/migration/upgrade/v3_4_0/PreflightCheckMigration.php

```diff
@@ -368,7 +449,7 @@ protected function checkForeignKeySupport(): void
         );
 
         if (count($result) > 0) {
-            error_log(print_r($result,true));
+            error_log(print_r($result, true));
```

Comment:
I've seen this before, and I forgot to remove it. I think it shouldn't be here.

Reply:
I can do it.. What exactly should or should not happen?

Reply:
Or to maybe open a new issue for that?

Reply:
I think this can be safely erased, someone was probably just debugging and forgot it here, the `$result` is used/displayed to the user below.